backend/src/main/java/com/bakdata/conquery/io/jetty/ConqueryErrorExceptionMapper.java

```diff
@@ -1,16 +1,20 @@
 package com.bakdata.conquery.io.jetty;
 
 import com.bakdata.conquery.models.error.ConqueryError;
+import com.bakdata.conquery.models.error.SimpleErrorInfo;
+
 import jakarta.ws.rs.core.MediaType;
 import jakarta.ws.rs.core.Response;
 import jakarta.ws.rs.ext.ExceptionMapper;
 
 public class ConqueryErrorExceptionMapper implements ExceptionMapper<ConqueryError> {
     @Override
     public Response toResponse(ConqueryError exception) {
-        return Response.status(Response.Status.BAD_REQUEST.getStatusCode())
-                .type(MediaType.APPLICATION_JSON_TYPE)
-                .entity(exception)
-                .build();
+		SimpleErrorInfo plain = exception.asPlain();
+
+		return Response.status(Response.Status.BAD_REQUEST.getStatusCode())
```

Comment:
wäre es möglich, dass du die Methode in einem test abdeckst?

Reply:
Ja, am sinnvollsten ist hier denke ich ein E2E test, denn der Sinn dieser Klasse ist es, ein Objekt zu erzeugen mit dem das Frontend ein spezifischeren Fehler anzeigt als "Anfrage fehlgeschlagen."

Ansonsten ist an dem Mapper denke ich nicht viel zu testen oder? Das meiste hängt ja dann doch Framework related und sollte von diesem abgetestet werden.

Reply:
@awildturtok Test ist jetzt hinzugefügt, ich musste auch noch im FE code eine kleine Änderung machen, damit nicht die generische Fehlermeldung angezeigt wird.

Reply:
![grafik](https://github.com/user-attachments/assets/4bd0d61e-72e4-4372-a286-bc20717e75f4)